src/rules/padding.ts

```diff
@@ -314,7 +316,7 @@ const verifyNode = (node: Node, paddingContext: PaddingContext): void => {
  */
 export const createRule = (
   configs: Config[],
-  deprecated: boolean = false,
+  deprecated = false,
```

Comment:
Why the removal of this type hint? Does it correctly infer that the type is `boolean`?

Reply:
The compiler (or dan, the human compiler, idk) probably flagged this as an unnecessary type annotation, because the type inference can definitely pick that one up.

Reply:
ESlint picked it up when I enabled it on TS. It does correctly infer that it is `boolean`

Reply:
Sweet. I was hoping that it would infer that but wanted to check. I was wondering if it would do something similar to strings with `string` vs `"concrete string variant 1" | "concrete string varient 2"`. I was wondering if it would infer that the type of this is `false` rather than `boolean`. Seems like no! :thumbsup: